docs/intro.rst

```diff
@@ -289,6 +302,12 @@ generating
 ``target/kafka-rest-$VERSION-standalone.jar``, which includes all the
 dependencies as well.
 
+Requirements
+------------
+
+- Kafka 0.9.0.0
```

Comment:
Should we explicitly set this to `0.9.0.0-cp1`?

That's what I did for Camus at https://github.com/confluentinc/camus/commit/68892fef86c7d019deeef3dde01b3b28b53543de, see the `docs/intro.rst` change.

My argument is that (1) this is good for consistency within CP and (2) this will ensure people will benefit from e.g. the producer regression fix that was added after 0.9.0.0 release.